### PR TITLE
Ref task 3028 show data on ui

### PR DESCRIPTION
**Pull request by manhphan34**
### Summary:
- [Feature] Profile screen
### Link Issue:
- https://edu-redmine.framgia.vn/issues/3028

### CODE REVIEW

**Comment by manhphan34** [comment]
ready

File: app/src/main/java/framgia/com/ichat/screen/onlineuser/OnlineUserAdapter.java [hunk 1]
```diff
@@ -82,6 +82,6 @@ public void onClick(View v) {
     }
 
     public interface onUserItemClickListener {
```
Comment (DoanVanToan) [inline app/src/main/java/framgia/com/ichat/screen/onlineuser/OnlineUserAdapter.java]:
```suggestion
    public interface OnUserItemClickListener {
```

**Comment by DoanVanToan** [comment]
commented

**Comment by manhphan34** [comment]
ready

**Review by DoanVanToan — APPROVED** [review]

**Pull request merged**